test/integration/targets/delegate_to/test_delegate_to.yml

```diff
@@ -1,9 +1,11 @@
 - hosts: testhost3
```

Comment:
wondering if there's a sane (cross platform-friendly) way to check for existence of the right IPs and skip this test if they're not present...